scene/gui/slider.cpp

```diff
@@ -68,14 +68,15 @@ void Slider::gui_input(const Ref<InputEvent> &p_event) {
 				double grab_height = (double)grabber->get_height();
 				double max = orientation == VERTICAL ? get_size().height - grab_height : get_size().width - grab_width;
 				if (orientation == VERTICAL) {
-					set_as_ratio(1 - (((double)grab.pos - (grab_height / 2.0)) / max));
+					set_as_ratio_no_signal(1 - (((double)grab.pos - (grab_height / 2.0)) / max));
```

Comment:
The method is internal, so I paid no attention, but technically you can do
```suggestion
					set_block_signals(true);
					set_as_ratio(1 - (((double)grab.pos - (grab_height / 2.0)) / max));
					set_block_signals(false);
```